doc.go

```diff
@@ -599,6 +599,25 @@ Note: As of February 2020, Golang's official library does not support passcode-e
 For security purposes, Snowflake highly recommends that you store the passcode-encrypted private key on the disk and
 decrypt the key in your application using a library you trust.
 
+# External browser authentication
+
+The driver allows to authenticate through the external browser.
```

Comment:
Is `through` the best word? What do you think about `using`?

Reply:
Good idea, done